samples/hello-terminal/todo-terminal-app/src/main/java/com/squareup/sample/hellotodo/TodoWorkflow.kt

```diff
@@ -76,14 +76,14 @@ class TodoWorkflow : TerminalWorkflow,
   override fun render(
     props: TerminalProps,
     state: TodoList,
-    context: RenderContext<TerminalProps, TodoList, ExitCode>
+    context: RenderContext
   ): TerminalRendering {
 
     context.runningWorker(props.keyStrokes) { onKeystroke(it) }
 
     return TerminalRendering(buildString {
       @Suppress("UNCHECKED_CAST")
-      appendln(state.renderTitle(props, context as RenderContext<TerminalProps, TodoList, Nothing>))
+      appendln(state.renderTitle(props, context))
```

Comment:
whoa

Reply:
This is actually unrelated, `renderTitle` should always have specified a wildcard rendering type.